server/src/main/resources/db/changelog/20210412145432-drop_content_cache.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20210412145432-1" author="ojanus" dbms="postgresql">
+        <sql>UPDATE cp_cert_serial SET expiration = NOW() FROM cp_cont_access_cert WHERE cp_cert_serial.id = cp_cont_access_cert.serial_id</sql>
+    </changeSet>
+
+    <changeSet id="20210412145432-2" author="ojanus" dbms="mysql,mariadb">
+        <sql>UPDATE cp_cert_serial s RIGHT JOIN cp_cont_access_cert c ON c.serial_id = s.id SET expiration = NOW()</sql>
+    </changeSet>
+
+    <changeSet id="20210412145432-3" author="ojanus">
+        <addColumn tableName="cp_cont_access_cert">
+            <column name="content" type="LONGTEXT">
+                <constraints nullable="false"/>
```

Comment:
While the port looks good, I thought I'd give the db upgrade a test, and found out there's a problem. I've:
1. Set up candlepin with test data
2. Switched one of the orgs to use SCA mode
3. Registered a consumer, so as to generate an SCA cert+serial
4. Checked out this branch and tried to deploy 
Got this error:
```
Updating Database
Liquibase update Failed: Migration failed for change set db/changelog/20210412145432-drop_content_cache.xml::20210412145432-3::ojanus:
     Reason: liquibase.exception.DatabaseException: Error executing SQL ALTER TABLE public.cp_cont_access_cert ADD content TEXT NOT NULL: ERROR: column "content" contains null values
```
It looks like when we're adding non-nullable columns, we need to define a default value for the existing rows in the table. Adding an empty string should do the job, since all sca certs are set to be expired, and next check-in will force the regeneration of their content payloads.

Reply:
Needless to say, we need to fix this also in 3.1 and master

Reply:
That sounds like something that needs to be fixed across all the branches, not just in the backport.